### PR TITLE
Add LUNA, UST metadata

### PR DESCRIPTION
**Pull request by grandicrypto**
# LUNA
ERC-20: https://etherscan.io/address/0xd2877702675e6ceb975b4a1dff9fb7baf4c91ea9
NEP-141: https://explorer.mainnet.near.org/accounts/d2877702675e6ceb975b4a1dff9fb7baf4c91ea9.factory.bridge.near

# UST
ERC-20: https://etherscan.io/address/0xa47c8bf37f92abed4a126bda807a7b7498661acd
NEP-141: https://explorer.mainnet.near.org/accounts/a47c8bf37f92abed4a126bda807a7b7498661acd.factory.bridge.near

### CODE REVIEW

File: tokens/ust.svg [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+<g><path style="opacity:0.999" fill="#5492f6" d="M 692.5,111.5 C 756.485,164.052 800.985,230.052 826,309.5C 829.33,321.157 832.497,332.824 835.5,344.5C 830.553,345.285 825.553,345.951 820.5,346.5C 726.526,361.961 632.526,377.295 538.5,392.5C 538.5,404.833 538.5,417.167 538.5,429.5C 538.5,442.167 538.5,454.833 538.5,467.5C 538.5,479.833 538.5,492.167 538.5,504.5C 538.5,516.833 538.5,529.167 538.5,541.5C 538.5,553.833 538.5,566.167 538.5,578.5C 538.5,591.167 538.5,603.833 538.5,616.5C 537.501,542.002 537.167,467.335 537.5,392.5C 506.836,397.074 476.17,401.407 445.5,405.5C 434.816,406.354 424.149,407.354 413.5,408.5C 393.365,410.52 373.365,412.52 353.5,414.5C 343.791,414.741 334.124,415.408 324.5,416.5C 307.845,417.44 291.178,418.107 274.5,418.5C 256.124,400.942 242.291,380.276 233,356.5C 223.144,326.421 225.644,297.421 240.5,269.5C 259.928,238.904 284.928,213.738 315.5,194C 380.149,153.521 449.815,124.854 524.5,108C 560.607,98.7461 597.274,94.7461 634.5,96C 654.918,97.4222 674.251,102.589 692.5,111.5 Z"/></g>
```
Comment (sept-en) [inline tokens/ust.svg:6]: The icon for the UST token is too huge, it seems it was converted from PNG, but it's too big to be set as metadata. Please find another SVG or make a conversion using another tool and simplifications (image size, number of colors, etc)

**Review by sept-en — CHANGES_REQUESTED** [review]

File: tokens/ust.svg (hunk 1 above)
Comment (grandicrypto) [reply tokens/ust.svg]: updated, please let me know if this size works!

Comment (sept-en) [reply tokens/ust.svg]: Thanks, still quite big, but works for now. But if you find a small (non-converted) SVG, please create another one PR.

**Review by sept-en — APPROVED** [review]

File: tokens/ust.svg (hunk 1 above)
Comment (grandicrypto) [reply tokens/ust.svg]: will do soon :+1:

**Pull request merged**